src/general/requiredlabels/src.cel

```diff
@@ -0,0 +1,5 @@
+validations:
+- expression: '(has(object.metadata) && variables.params.labels.all(entry, has(object.metadata.labels) && entry.key in object.metadata.labels))'
+  messageExpression: '"missing required label, requires all of: " + variables.params.labels.map(entry, entry.key).join(", ")'
+- expression: '(has(object.metadata) && !variables.params.labels.all(entry, has(object.metadata.labels) && entry.key in object.metadata.labels && !string(object.metadata.labels[entry.key]).matches(string(entry.allowedRegex))))'
```

Comment:
Can you please test the regex use case? Maybe even add a test for it? 

> !string(object.metadata.labels[entry.key]).matches(string(entry.allowedRegex)

This feels incorrect...it should match on the allowedRegex not `!`. Please test and verify this. The example in GK might also be wrong.

Reply:
@ritazh I think we need it to `not match` on the allowedRegex. If value do not match the `allowedRegex` then its a violation otherwise it is not.

Reply:
I added the example testing the same.

Reply:
I missed `!variable` :)

Reply:
@ritazh I changed the CEL source. Let me know what you think.